packages/css_colors/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: css_colors
 description: Defines constant dart:ui Color objects for CSS colors (for use in Flutter code).
 repository: https://github.com/flutter/packages/tree/main/packages/css_colors
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+css_colors%22
-version: 1.1.4
+version: 1.1.4+1
```

Comment:
Please see the end of https://dart.dev/tools/pub/versioning#semantic-versions. We follow the advice there about when to use (and not use) `+` in versions.